Gaijinizer patch/Map070.txt

```diff
@@ -2119,29 +2187,31 @@ spirit of Haru.#MTLed
 
 そう不安になりつつも、彼女に任せてみたんだ。
 > CONTEXT: Map070/events/9/pages/1/248/Dialogue/enemy7
-"This is because I'm only living at home
-You don't know human fear? ". 
-Even though I became uneasy, I tried to leave it
-to her.#MTLed
+I thought that the only reason she could propose
+such a thing was because as a housecat, she had
+no concept of how awful humans could be. But
+despite my reservations, I went along with it.#Ron
```

Comment:
Let's move this in two textboxes so we don't have to cut off the text...
>_**I thought that the only reason she would
propose such a thing was that as a housecat,
she had no idea of how awful humans could be.**_
>
>_**But despite my reservations, I went
along with it.**_